### PR TITLE
feat: Add version endpoint

### PR DESCRIPTION
**Pull request by holzeis**
Demonstrates a simple cut through from the web ui to the backend.

### CODE REVIEW

File: webapp/frontend/lib/common/version_service.dart [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  }
+}
+
+class VersionService {
+  const VersionService();
+
+  Future<String> fetchVersion() async {
+    // TODO(holzeis): this should come from the config
+    const port = "3001";
+    const host = "localhost";
```
Comment (bonomat) [inline webapp/frontend/lib/common/version_service.dart:26]: how do you see this evolve to support relative paths from where it's being hosted from?

File: webapp/src/main.rs [hunk 2]
```diff
@@ -120,4 +124,4 @@ async fn serve(app: Router, port: u16) -> Result<()> {
     tracing::debug!("listening on {}", listener.local_addr()?);
     axum::serve(listener, app).await?;
     Ok(())
-}
+}
```
Comment (bonomat) [inline webapp/src/main.rs:127]: 🙃
Missing new line

**Review by bonomat — APPROVED** [review]
Nice

File: webapp/frontend/lib/common/version_service.dart (hunk 1 above)
Comment (holzeis) [reply webapp/frontend/lib/common/version_service.dart:26]: I fear we can't the URI path argument required by the http package does not support relative paths. At least I haven't found a way of doing that. I presume we will have to set that in a config. (see todo).

**Pull request merged**